### PR TITLE
Preparations for Release 24-06

### PR DESCRIPTION
**Pull request by javfg**

### CODE REVIEW

File: profiles/config.dev-ppp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # PIS config
-OTOPS_EFO_RELEASE_VERSION=v3.62.0
+OTOPS_EFO_RELEASE_VERSION=v3.65.0
 OTOPS_ENSEMBL_RELEASE_VERSION=111
 OTOPS_CHEMBL_RELEASE_VERSION=34
 OTOPS_PIS_RELEASE_VERSION=24.06dev
```
Comment (mbdebian) [inline profiles/config.dev-ppp:5]: I would change this as well, to 24.06

**Review by mbdebian — CHANGES_REQUESTED** [review]
Just a minor change request

File: profiles/config.dev-ppp (hunk 1 above)
Comment (javfg) [reply profiles/config.dev-ppp:5]: As we discussed, we're leaving it there and removing it in a profile dedicated to 24-06 release.

**Review by mbdebian — APPROVED** [review]
🙌🏻 thanks!

**Pull request merged**